homeworks/Vladimir.Karbivnychiy_VladimirSkarb/8-components-templates/pictures-slider/src/app/interfaces/dataInterface.ts

```diff
@@ -0,0 +1,10 @@
+import { UrlsInterface } from './urlsInterface';
+import { LinksInterface } from './linksInterface';
+
+export interface DataInterface {
```

Comment:
1) You don't need to add the `Interface` postfix for each interface you create

2) `Data` name is too abstract. `Image` name would be better and easier to understand

3) Move all your image related interfaces into the one file 

**Applicable to all interfaces**